src/service/CallApi.js

```diff
@@ -49,4 +49,12 @@ export default class CallApi {
   static async relocateCall(callId: string, line: number, contactIdentifier?: string): Promise<Relocation> {
     return getApiClient().ctidNg.relocateCall(callId, 'line', line, contactIdentifier);
   }
+
+  static async mute(callId: string): Promise<void> {
+    return getApiClient().ctidNg.mute(callId);
+  }
+
+  static async unmute(callId: string): Promise<void> {
+    return getApiClient().ctidNg.unmute(callId);
```

Comment:
can you check with @sboily but I think that ctidNg should be replaced by calld.

Reply:
This has implications larger than the scope of this PR, mind if we "shovel this one forward" to paraphrase an old Québecois saying?

Reply:
Yes we need to use calld instead ctidNg. I have duplicated ctidNg with calld to permit this migration.

Reply:
Yes we need to use calld.